.github/workflows/prepare_post_release.yml

```diff
@@ -52,14 +52,14 @@ jobs:
             ${{ github.repository }} \
             ${{ steps.pipeline_context.outputs.candidate_tag }}
       - name: "Create patch branch"
-        if: github.ref_name == 'master'
+        if: github.ref_name == 'main'
```

Comment:
Do we need to backport any of these workflow changes to old versions branches? I'm assuming each branch will use the file they contain to spin up GitHub workflows but I'm not sure.

Reply:
Good point. It won't make a difference, as those branches are not called either `master` or `main`, but we can backport the changes for consistency.